avaframe/runAna4ProbAna.py

```diff
@@ -54,7 +54,7 @@ def runProbAna(avalancheDir=''):
     log.info('Current avalanche: %s', avalancheDir)
 
     # Clean input directory(ies) of old work files
-    initProj.cleanSingleAvaDir(avalancheDir, keep=logName, deleteOutput=False)
+    initProj.cleanSingleAvaDir(avalancheDir, deleteOutput=False)
```

Comment:
no option to keep log anymore?

Reply:
Nope. If you delete the output -> logs are gone. If you keep the output -> logs are kept